### DISCUSSION TITLE
Complete the expression

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1121] [Galaxy Trivia] [answered]
> Some day you will be old enough ...

**Bonus:** Name the author.

**Comment by LeleDallas** [comment, marked as answer]
To start reading fairy tales again - C.S. Lewis

**Comment by peterbaccaro** [comment]
The complete expression is, **"Someday you will be old enough to start reading fairy tales again."**.
This quote is often attributed to **C.S. Lewis**, and it reflects the idea that no matter how old we get, there's always room for wonder and imagination, including enjoying stories like fairy tales.
